### DISCUSSION TITLE
Addons at the environment level

### DISCUSSION THREAD

**Discussion by rverpillot** [aws/copilot-cli#4039] [Q&A] [answered]
Now we have environment manifests, can we use addons at the environment level ? I can't find anything on this topic in the documentation.

**Comment by efekarakus** [comment]
The design is coming! @Lou1415926 is working on it, once it's ready we'll pin a GitHub issue and reply here.

**Comment by mvn-bachhuynh-dn** [comment]
Thank you @rverpillot , It's my problem now when I try to delete a service and included add-on deleted too...
Hope Copilot could use add-on on env manifest soon! <3

**Comment by efekarakus** [comment, marked as answer]
We forgot to reply here 🙇 , but here is the design for Environment Addons! It's in active development now: https://github.com/aws/copilot-cli/issues/4219
